exec-unittests

```diff
@@ -157,49 +129,11 @@ if [[ -z "${subtst}" || "${subtst}" == "python" ]]; then
     fi
 fi
 
-_subtst_list="tool-scripts/datalog tool-scripts/postprocess tool-scripts util-scripts bench-scripts"
-
-_para_jobs_file="${_toxenvdir}/agent-legacy-jobs.lis"
-trap "rm -f ${_para_jobs_file}" EXIT INT TERM
-
-let count=0
 for _major in ${major_list}; do
     # Verify the Agent or Server Makefile functions correctly.
     if [[ "${_major}" == "agent" || "${_major}" == "server" ]]; then
         verify_make_source_tree ${_major} || rc=1
     fi
-
-    if [[ "${_major}" == "agent" ]]; then
-        # The parallel program is really cool.  The usage of `parallel` is
-        # internal and automated; only test code depends on this tool, and we,
-        # as developers, have viewed the citation and are justified in
-        # suppressing future displays of it in our development processes (use of
-        # --will-cite below).
-
-        for _subtst in ${_subtst_list}; do
-            if [[ "${subtst:-legacy}" == "legacy" || "${subtst}" == "$(basename ${_subtst})" ]]; then
-                echo "${_major} ${_subtst} ${posargs}"
-                (( count++ ))
-            fi
-        done > ${_para_jobs_file}
-        parallel --will-cite -k --lb -a ${_para_jobs_file} ${para_jobs_arg} para_run_legacy
-        if [[ ${?} -ne 0 ]]; then
-            rc=1
-        fi
-    elif [[ "${_major}" == "server" ]]; then
-        if [[ -z "${subtst}" || "${subtst}" == "legacy" ]]; then
-            (( count++ ))
-            run_legacy ${_major} bin ${posargs} || rc=1
-        fi
-    elif [[ "${_major}" != "client" ]]; then
-        printf -- "Error - unrecognized major test sub-set, '%s'\n" "${_major}" >&2
-        rc=1
-    fi
```

Comment:
Maybe don't delete these lines, and replace line 170 with them?

Reply:
That check was already really redundant with line 21. We know that `_major` can only be `agent`, `server`, or `client`. Somehow this logic felt appropriate when I was adding `client` unit tests, but it really wasn't necessary then and doesn't make much sense now.

Reply:
Quite right.  Thanks for pointing out line 21.  Nevermind.